Doc/library/pickle.rst

```diff
@@ -428,6 +435,34 @@ The :mod:`pickle` module exports two classes, :class:`Pickler` and
       :ref:`pickle-restrict` for details.
 
 
+.. class:: PickleBuffer(buffer)
+
+   A wrapper for a potentially out-of-band buffer.  *buffer* must be a
```

Comment:
I'm not sure what "out-of-band" means. I found https://en.wikipedia.org/wiki/Out-of-band_data

Maybe add a link to the "Out-of-band Buffers" section below? This section explains well what you mean :-)